src/wp-content/themes/twentytwentyone/style.css

```diff
@@ -2624,6 +2624,59 @@ ol {
 	margin: 0;
 	padding-left: calc(2 * var(--global--spacing-horizontal));
 }
+/* Lower Alpha */
```

Comment:
The twentytwentyone theme use SASS for CSS so you have to update SASS file instead of actual CSS https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-content/themes/twentytwentyone/assets/sass/05-blocks/list/_style.scss. It will auto generate CSS file. 

cc. @carolinan

Reply:
thank you for findings